src/routers/bookmark_router.py

```diff
@@ -0,0 +1,140 @@
+from fastapi import APIRouter, Depends, HTTPException
+from typing import List
+from pydantic import BaseModel
+from sqlmodel import Session, select
+
+from authentication import KeycloakUser, get_user_or_raise
+from database.session import get_session
+from database.model.bookmark.bookmark import Bookmark
+from http import HTTPStatus
+from database.model.concept.concept import AIoDConcept
+from database.model.helper_functions import non_abstract_subclasses
+
+
+class BookmarkCreate(BaseModel):
+    resource_identifier: str
+
+
+class BookmarkRead(BaseModel):
+    resource_identifier: str
+    created_at: str
+
+
+def create(url_prefix: str = "") -> APIRouter:
+    router = APIRouter()
+    version = "v1"
+
+    for path in [
+        f"{url_prefix}/bookmarks/{version}",
+        f"{url_prefix}/v2/bookmarks",
+        f"{url_prefix}/bookmarks",
+    ]:
+
+        @router.get(
+            path,
+            tags=["Bookmarks"],
```

Comment:
```suggestion
            tags=["User"],
```
Let's put this in the same category with 'my resources' for general user-related endpoints.